packaging/tags.py

```diff
@@ -108,6 +112,12 @@ def __repr__(self):
 
 def parse_tag(tag):
     # type: (str) -> FrozenSet[Tag]
+    """
+    Parses the provided tag (e.g. `py3-none-any`) into a set of Tag instances.
```

Comment:
Might as well be specific about the return type so people know they can rely on it.
```suggestion
    Parses the provided tag (e.g. `py3-none-any`) into a frozenset of Tag instances.
```